## Problem 4.6

### 4.6.a
We know that in a binary system,
\begin{align}
r_1 =& \dfrac{M_1}{M_1+M_2} a = \frac{M}{M+M} a\\
\implies r_1 =& \frac{a}{2} \\
r_1 + r_2 =& a\\
\implies r_1 =& r_2 = a/2
\end{align}
The reducded mass of the system is 
\begin{align}
\mu = \frac{M_1 M_2}{M_1+M_2} = \frac{M}{2}
\end{align}
The orbital velocity can be obtained by equating the force of gravity with the centrifugal force
\begin{align}
\frac{Mv^2}{r_1} =& \frac{G\mu^2}{r_1^2}\\
\implies v^2 =& \frac{GM}{2a}
\end{align}
Now, we can obtain the total orbital kinetic energy as
\begin{align}
E_K = \frac{1}{2}Mv^2 + \frac{1}{2}Mv^2 = \frac{GM}{2a}
\end{align}
The gravitational energy and hence, the total energy in the system is
\begin{align}
E_{gr} = -\frac{GM^2}{a} \qquad E_{tot} = -\frac{GM^2}{2a}
\end{align}

### 4.6.b
Power lost to gravitational energy
\begin{align}
\dot{E}_{gw} = -\frac{2c^5}{5G}\Big(\frac{2GM}{c^2a}\Big)^5
\end{align}
The change in the total energy is given by
\begin{align}
\dot{E}_{tot} = \frac{d}{dt}\Big(-\frac{GM^2}{2a}\Big) = \frac{GM^2}{2a^2} \frac{da(t)}{dt}
\end{align}
This gives us
\begin{align}
\frac{GM^2}{2a^2} \frac{da(t)}{dt} =& -\frac{2c^5}{5G}\Big(\frac{2GM}{c^2a}\Big)^5\\
\therefore a^3\frac{da(t)}{dt} =& -\frac{4c^5}{5G^2M^2}\Big(\frac{2GM}{c^2}\Big)^5 = -\frac{128}{5}\frac{G^3M^3}{c^5}
\end{align}
Solving for a, we get
\begin{align}
a^3\frac da = -\frac{128}{5}\frac{G^3M^3}{c^5} dt\\
\implies \frac{a^4}{4}\Big|^{a=a(t)}_{a=a_0} = -\frac{128}{5}\frac{G^3M^3t}{c^5}
\end{align}

### 4.6.c
Given t = 10Gyr, $M=1M_\odot$ and a(t)=0;
\begin{align}
a_0^4 = \frac{4\times 128}{5} \frac{G^3M^3 10^{10}yr}{c^5}
\end{align}

In [3]:
a0 = (4*128*(6.67e-8**3)*(2e33**3)*1e10*365*24*3600/(3e10**5))**0.25 #cm
a0AU = a0*6.68e-14
print(a0,a0AU)

354392437505.26843 0.023673414825351933


\begin{align}
\therefore a_0 = 0.0237 AU
\end{align}

## Problem 4.8

### 4.8.a
\begin{align}
\frac{F_{tide}}{F_{grav}} = \frac{2M_2}{M_1} \Big(\frac{\Delta r}{r}\Big)^3
\end{align}

In [4]:
m_moon = 7.35e25
m_earth = 5.97e27
r_earth = 6.378e8
d_moon = 3.84e10
ratio = 2*m_moon*((r_earth/d_moon)**3)/m_earth
print(ratio)

1.1282435911360697e-07


\begin{align}
\therefore \frac{F_{tide}}{F_{grav}} = 1.128 \times 10^{-7}
\end{align}

Given that gR = constant on earth's surface
\begin{align}
\Delta g R + g\Delta R =& 0\\
\implies \Delta R =& \Big|-\frac{\Delta g}{g} R\Big| = \frac{F_{tide}}{F_{grav}}R 
\end{align}

In [5]:
dr = ratio*r_earth
print(dr)

71.95937624265852


\begin{align}
\therefore \Delta R = 71.95 cm
\end{align}

For the sun, 

In [8]:
m_sun = 2e33
d_sun = 1.49e13
ratio = 2*m_sun*((r_earth/d_sun)**3)/m_earth
print(ratio)

5.2550929212234943e-08


In [9]:
dr = ratio*r_earth
print(dr)

33.516982651563445


\begin{align}
\therefore \Delta R = 33.52cm
\end{align}

### 4.8.b
The ratio of the forces gives us the ratio of masses during eclipse as
\begin{align}
\Delta m = \frac{F_{tide}}{F_{grav}} M
\end{align}

In [10]:
ratio = 1.128e-7*50e3
print(ratio)

0.00564


\begin{align}
\therefore \Delta m = 0.0056g = 5.6 mg
\end{align}

### 4.8.c
The tidal stretch is simply given by the tidal force exerted by the moon on my extended body of length $\Delta R$ as
\begin{align}
F = \frac{2GM_{moon}m\Delta R}{R^3}
\end{align}

In [13]:
force = 2*6.67e-8*m_moon*50e3*180/(d_moon**3)
gforce = force/980
print(force,gforce)

1.5584468841552734e-06 1.5902519226074218e-09


\begin{align}
\therefore \textrm{stretch} = 1.55\times 10^{-6} dyne = 1.59\times 10^{-9} \textrm{gram force}
\end{align}